Make.coverity

```diff
@@ -0,0 +1,48 @@
+COV_EMAIL=$(call get-config,coverity.email)
+COV_TOKEN=$(call get-config,coverity.token)
+COV_URL=$(call get-config,coverity.url)
+COV_FILE=$(NAME)-coverity-$(VERSION)-$(COMMIT_ID).tar.bz2
+
+cov-int : clean-shim-objs
+	make $(DASHJ) Cryptlib/OpenSSL/libopenssl.a Cryptlib/libcryptlib.a
+	cov-build --dir cov-int make $(DASHJ) all
+
+cov-int-all : clean
+	cov-build --dir cov-int make $(DASHJ) all
+
+cov-clean :
+	@rm -vf $(NAME)-coverity-*.tar.*
```

Comment:
Shouldn't `@rm -vf $(COV_FILE)` be enough? I think that makes it easier to follow.

Reply:
That doesn't work as well - it means if you do two runs with an intervening commit and forget to make clean, one of them isn't cleaned up.